.github/workflows/build-test.yaml

```diff
@@ -92,6 +92,7 @@ jobs:
         token: ${{ secrets.PERSONAL_TOKEN }}
         wait-for-completion-interval: 10s
         wait-for-completion-timeout: 5m
+        repost-logs: true
```

Comment:
Can you rename the parameter to `print-workflow-logs` or better for future improvements `workflow-logs` that can take a string/enum value such as `print`. So that in a later version, it could send logs somewhere else.